doc/howto/json.md

```diff
@@ -86,12 +86,17 @@ for d in data:
     lines += json.dumps(d) + "\n"
 
 _ = Path("people.jsonl").write_text(lines)
```

Comment:
`_ = Path("people.jsonl").write_text(lines)` -> `_ = Path("people.json").write_text(lines)` ?

Reply:
I think we wanted to still demonstrate some functionality for `.jsonl` files

Reply:
yes, we want to demonstrate both json, and jsonl - let's keep it as is